symplyphysics/laws/hydro/velocity_of_liquid_flowing_out_of_pipuid_flowing_out_of_pipe.py

```diff
@@ -0,0 +1,40 @@
+from sympy import (Eq, solve, sqrt)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Consider a tube through which a liquid flows under arbitrary pressure. Then the velocity of the
+## liquid flowing out of the pipe will depend only on the pressure and density of the liquid.
+
+## Law is: v = sqrt(2 * p / p0), where
+## v - velocity of the liquid flowing out of the pipe,
+## p - pressure,
+## p0 - density of the liquid.
+
+velocity = Symbol("velocity", units.velocity)
+
+pressure = Symbol("pressure", units.pressure)
+density = Symbol("density", units.mass / units.volume)
+
+law = Eq(velocity, sqrt( 2 * pressure / density))
```

Comment:
Add proof using https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/hydro/velocity_from_height.py

Reply:
> Add proof using https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/hydro/velocity_from_height.py

As I understand it, my law cannot be derived from the above.
The law "velocity_from_height" is derived from Bernoulli's law for a special case (the body is filled with liquid and there is a hole in the body).
In the same law, we are talking about a tube located horizontally. I can get the velocity from Bernoulli's equation, but it is not among the laws.